app/serializers/lib_search_result_serializer.rb

```diff
@@ -0,0 +1,21 @@
+class LibSearchResultSerializer < ObjectSerializer
+  set_id { |object| Fingerprintable::Fingerprinter.new(object: object).fingerprint }
+
+  attributes :pagination, :query
```

Comment:
When I hit the API locally, I noticed the `pagination` param is empty
```
      "pagination": {},
```
Is that intentional/is it needed?

Reply:
Yes, see the PR comment.